docs/create-a-pipeline.md

```diff
@@ -95,10 +95,12 @@ register_pipeline(
 Finally add this at the bottom of your file to start the app:
 
 ```py title="src/app.py"
+app = get_app()
+
 if __name__ == "__main__":
     import uvicorn
 
-    uvicorn.run("plombery:get_app", reload=True, factory=True)
+    uvicorn.run(app, reload=True, factory=True)
```

Comment:
If you provide an `app` object to uvicorn, then you need to remove the `factory=True`, otherwise uvicorn will call the first argument, i.e. `app()` expecting an app object in return.

BTW why you replaced the original uvicorn line with this approach? In general, I decided to use the factory method so uvicorn is responsible for instantiating the app when needed and possibly we could add some init logic inside get_app function.

Reply:
Fixed, thanks :)

I am invoking uvicorn from command line, with custom parameters, such as `--host 0.0.0.0`. Changing the invocation is way quicker than changing the code when working with container (just run the same image with a different command).

Reply:
`WARNING:  You must pass the application as an import string to enable 'reload' or 'workers'.`

So to use both main.py directly and uvicorn invocation, we need both app and the string.